stubs/braintree/braintree/configuration.pyi

```diff
@@ -1,45 +1,70 @@
 from _typeshed import Incomplete
+from typing import Literal
+
+from braintree.braintree_gateway import BraintreeGateway
+from braintree.util.graphql_client import GraphQLClient
+from braintree.util.http import Http
 
 class Configuration:
     @staticmethod
-    def configure(environment, merchant_id, public_key, private_key, **kwargs) -> None: ...
+    def configure(
+        environment,
+        merchant_id: str,
+        public_key: str,
+        private_key: str,
+        *,
+        http_strategy: Incomplete | None = None,
+        timeout: int = 60,
+        wrap_http_exceptions: bool = False,
+    ) -> None: ...
     @staticmethod
-    def for_partner(environment, partner_id, public_key, private_key, **kwargs): ...
+    def for_partner(
+        environment,
+        partner_id: str,
+        public_key: str,
+        private_key: str,
+        *,
+        http_strategy: Incomplete | None = None,
+        timeout: int = 60,
+        wrap_http_exceptions: bool = False,
+    ) -> Configuration: ...
     @staticmethod
-    def gateway(): ...
+    def gateway() -> BraintreeGateway: ...
     @staticmethod
-    def instantiate(): ...
+    def instantiate() -> Configuration: ...
     @staticmethod
-    def api_version(): ...
+    def api_version() -> Literal["6"]: ...
```

Comment:
I wouldn't hard-code the return values here. It's conceivable that they change in the future and I don't think callers should rely on the exact value here.

```suggestion
    def api_version() -> str: ...
```